packages/lib-classifier/src/store/Classification/Classification.js

```diff
@@ -27,9 +27,15 @@ const Classification = types
       return snapshot
     },
 
-    get interactionTaskAnnotations () {
-      const annotations = Array.from(self.annotations.values()) || []
-      return annotations.filter(annotation => (getType(annotation).name === 'DrawingAnnotation' || getType(annotation).name === 'TranscriptionAnnotation'))
+    previousInteractionTaskAnnotations (activeTaskKey) {
+      if (activeTaskKey) {
+        const annotations = Array.from(self.annotations.values()) || []
+        const interactionTaskAnnotations = annotations.filter(annotation => (getType(annotation).name === 'DrawingAnnotation' || getType(annotation).name === 'TranscriptionAnnotation'))
+        return interactionTaskAnnotations.filter(annotation => annotation.task !== activeTaskKey) 
+      }
+
+      if (console) console.warn('Did you mean to call previousInteractionTaskAnnotations without an active task key?')
+      return []
```

Comment:
This will remove previously drawn marks on any step that doesn’t include a drawing task or transcription task.